packages/foreman/foreman/foreman.service

```diff
@@ -5,10 +5,10 @@ After=network.target remote-fs.target nss-lookup.target
 
 [Service]
 Type=simple
-User=foreman
+User=$FOREMAN_USER
 TimeoutSec=300
 WorkingDirectory=/usr/share/foreman
-ExecStart=/usr/bin/scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT"
+ExecStart=/usr/bin/scl enable tfm "rails server -e $FOREMAN_ENV -p $FOREMAN_PORT -b $FOREMAN_BIND"
 Environment=FOREMAN_ENV=production FOREMAN_PORT=3000
```

Comment:
Don't we need to set defaults here for the new variables? Currently the installer doesn't generate these so users are likely to run into issues. It also lists the sysconfig as optional

Reply:
Should we just make sysconfig required? Seems cleaner that way.

Reply:
IMHO we can't rely on the content set in sysconfig. In fact, systemd recommends removing these sysconfig files entirely and moving to systemd drop in files to override values.